sw/airborne/autopilot.c

```diff
@@ -300,20 +291,6 @@ bool autopilot_in_flight(void)
   return autopilot.in_flight;
 }
 
-/** set power switch
- */
-void autopilot_set_power_switch(bool power_switch)
```

Comment:
Please, can you make a separated module from this instead of completely removing it ?

Reply:
Will do that

Reply:
Done!